ibm/service/vpc/resource_ibm_is_volume.go

```diff
@@ -27,7 +26,7 @@ const (
 	isVolumeCapacity              = "capacity"
 	isVolumeIops                  = "iops"
 	isVolumeCrn                   = "crn"
-	isVolumeTags                  = "tags"
```

Comment:
This will break all existing customers ...We already define tags for all VPC resources and setting using global tagging API...Then follow th standard procedure of Terraform process
https://www.terraform.io/plugin/sdkv2/best-practices/deprecations

Reply:
done.